docker-compose.yaml

```diff
@@ -18,7 +18,7 @@
 #
 services:
   hive:
-    image: apache/gravitino-playground:hive-2.7.3
+    image: apache/gravitino-playground:hive-2.7.3.1
```

Comment:
Why do we need to add a new ".1" version, what is the difference? Also, I don't see we updated the doc about this version, can you also update the doc?

Reply:
The new Hive image supported to use Ranger. cc @yuqi1129 Could you help me update the document?